com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -1,8 +1,8 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
This is still here because of profiler stuff (which is really broken and needs a separate pass). It's now at the right position at least 🙃